src/Pages/Playlist/Playlist.jsx

```diff
@@ -1,11 +1,75 @@
-import React from 'react'
+import React from "react";
+import { useEffect } from "react";
+import axios from "axios";
+import { usePlay } from "../../context/playListContext";
+import { getUserToken } from "../../Utils/getUserToken";
+import { VideoCard } from "../../Components/VideoCard/VideoCard";
+import "./Playlist.css";
+import deletePlayList from "../../Utils/deletePlayList";
+import { useNavigate } from "react-router-dom";
```

Comment:
You can sort all the default imports at the top and then manual imports at the bottom. 
```
import React, { useEffect } from 'react';

 import axios from "axios";

import { useNavigate } from "react-router-dom";

import { usePlay } from "../../context/playListContext";

import { VideoCard } from "../../Components/VideoCard/VideoCard";

import { getUserToken } from "../../Utils/getUserToken";

import deletePlayList from "../../Utils/deletePlayList";

import "./Playlist.css";

```